src/models/Component.js

```diff
@@ -155,6 +155,39 @@ class Component extends FileInformation {
         return true;
       });
   }
+
+  /**
+   * Get attribute by name.
+   * @param {String} name - Name of attribute to find.
+   * @returns {ComponentAttribute|null} Component attribute or null.
+   */
+  getAttributeByName(name) {
+    return this.__getAttributeByName(this.attributes, name);
+  }
+
+  /**
+   * Get attribute from attributes list by name.
+   * @param {ComponentAttribute[]} attributes - Attributes list.
+   * @param {String} name - Name of attribute to find.
+   * @returns {ComponentAttribute|null} Component attribute or null.
+   * @private
+   */
+  __getAttributeByName(attributes, name) {
+    for (let index = 0; index < attributes.length; index += 1) { // NOSONAR
+      if (attributes[index].name === name) {
+        return attributes[index];
+      }
+      if (attributes[index].type === 'Object') {
+        const attribute = this.__getAttributeByName(attributes[index].value, name);
```

Comment:
What happens if multiple Object type Components have similarly named sub attributes?

Reply:
Question ask by @k-y0u  too and I say we don't still defined the behavior for sub-attributes, but we still want to manage link in object attribute. So for now we just get the first attribute with the wanted name as result.

It will be improve in a next version.